cirq-ionq/cirq_ionq/ionq_native_target_gateset.py

```diff
@@ -121,13 +121,7 @@ def _hadamard(self, qubit):
         return [GPI2Gate(phi=0.25).on(qubit), GPIGate(phi=0).on(qubit)]
 
     def _cnot(self, *qubits):
-        return [
-            GPI2Gate(phi=1 / 4).on(qubits[0]),
-            MSGate(phi0=0, phi1=0).on(qubits[0], qubits[1]),
-            GPI2Gate(phi=1 / 2).on(qubits[1]),
-            GPI2Gate(phi=1 / 2).on(qubits[0]),
-            GPI2Gate(phi=-1 / 4).on(qubits[0]),
-        ]
+        return NotImplemented()
```

Comment:
`NotImplemented` is not callable - it is a constant with special meaning for operator methods (e.g., `__eq__`).
The correct thing here is to **raise** the `NotImplementedError` exception (a different thing to the `NotImplemented` constant).

Please see details here - https://docs.python.org/3.10/library/constants.html?highlight=notimplemented#NotImplemented


```suggestion
        raise NotImplementedError()
```